challenges/classes.js

```diff
@@ -1,11 +1,46 @@
 // Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMaker {
+  constructor(dimensions) {
+    this.length = dimensions.length;
+    this.width = dimensions.width;
+    this.height = dimensions.height;
+  }
+
+  volume() {
+    return this.length * this.height * this.width;
+  }
+
+  surfaceArea() {
+    return (
+      2 *
+      (this.length * this.width +
+        this.length * this.height +
+        this.width * this.height)
+    );
+  }
+}
+
+const cuboid = new CuboidMaker({ length: 4, width: 5, height: 5 });
+
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+console.log(cuboid.volume()); // 100
+console.log(cuboid.surfaceArea()); // 130
 
 /* Stretch Task:
 
   Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods as well.  Create a new cube object and log out the results of your new cube.
 
-*/
\ No newline at end of file
+*/
+
+class CubeMaker extends CuboidMaker {
+  constructor(side) {
+    // super is giving me access to the methods of CuboidMaker and forcing the "side" measurement to populate the "length" "width" and "height" keys to make the methods work
+    super({ length: side, width: side, height: side });
```

Comment:
Great job using what you already created to calculate both the volume and surface area!